### PR TITLE
[ContactStructuralMechanicsApplications] Adding Exception when running with sympy>=1.3

### PR DESCRIPTION
**Pull request by loumalouomega**
Additionally adding documentation. Temp. fix

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/python_scripts/custom_sympy_fe_utilities.py [hunk 1]
```diff
@@ -1,28 +1,66 @@
...
+
+def DefineMatrix( name, m, n ):
+    """ This method defines a symbolic matrix
+
+    Keyword arguments:
+    name -- Name of variables.
+    m -- Number of rows.
+    n -- Number of columns.
+    """
+    if float(GetSympyVersion()) <= 1.2:
```
Comment (RiccardoRossi) [inline applications/ContactStructuralMechanicsApplication/python_scripts/custom_sympy_fe_utilities.py:16]: @loumalouomega this works just fine as long as you use it just as a symbol no?

it fails when you need more complex behaviour but not for most of simple usages

**Review by RiccardoRossi — CHANGES_REQUESTED** [review]
i think this is only needed when you try to use it as a function...

File: applications/ContactStructuralMechanicsApplication/python_scripts/custom_sympy_fe_utilities.py (hunk 1 above)
Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/python_scripts/custom_sympy_fe_utilities.py:16]: Yes, I am working in the actual solution. Can you approve?

**Comment by RiccardoRossi** [comment]
ok this is for the contact alone. sorry about the noise

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**